test/release/examples/benchmarks/shootout/mandelbrot-fast.prediff

```diff
@@ -0,0 +1,35 @@
+#!/bin/bash
```

Comment:
I think our "preferred" scheme is /usr/bin/env. tbh, it probably doesn't really matter, but I tend to go with /usr/bin/env for everything